expertsystem/amplitude/canonicaldecay.py

```diff
@@ -73,6 +76,73 @@ def generate_unique_amplitude_name(self, graph, node_id=None):
         return name
 
 
+def _clebsch_gordan_decorator(decay_generate_function):
```

Comment:
what is that move good for?

Reply:
`flake8` was complaining this is no method, but a function. First I changed it into a `staticmethod`, but then it's not a proper decorator anymore